docs/aurora/known-issues.md

```diff
@@ -101,8 +101,6 @@ This is a collection of known issues that have been encountered during Aurora's
 
 A known issues [page](https://apps.cels.anl.gov/confluence/display/inteldga/Known+Issues) can be found in the CELS Wiki space used for NDA content. Note that this page requires a JLSE Aurora early hardware/softare resource account for access.
 
-### Runtime Errors
```

Comment:
was this removal intentional? It will break the right-hand side ToC if there is a missing heading level in the hierarchy